torchgen/gen.py

```diff
@@ -1638,6 +1694,7 @@ def gen_per_operator_headers(
                         cpp_namespace="at::native",
                         class_method_name=None,
                         skip_dispatcher_op_registration=False,
+                        composite_graph={},
```

Comment:
It might be the case that the composite graph is only needed in the `ANONYMOUS_DEFINITION` case, but I think it could be confusing to pass an empty dict in all the other cases. Maybe either pass the composite graph in every case, or change the type to `Optional[CompositeGraph]` (to indicate that it won't be used in some cases?)

Reply:
That sounds better, indeed. But shouldn't be necessary with the new changes.